EIPS/eip-689.md

```diff
@@ -38,11 +39,21 @@ It seems impractical to implement never-used features just for passing tests.  C
 
 This EIP is backwards compatible on the main network.
 
+## Acknowledgements
+
+This has been previously suggested as [EIP-684](https://github.com/ethereum/EIPs/issues/684) and [EIP-83](https://github.com/ethereum/EIPs/issues/83).
+
```

Comment:
I'm not a fan of Acknowledgements section, I don't think it belongs in a technical specification (especially a CC0 licensed one).  I don't want this to turn into some sort of political/status thing where people cross-link to each other and give shout outs and kudos like we see in academia and at awards ceremonies.